source/python/deps/_tests/test_deps_cli.py

```diff
@@ -182,32 +191,38 @@ def test_cant_find_root(cli_runner, project_tree, piped_shell_execute):
     )
 
 
-def test_execution_on_project_dir(cli_runner, project_tree, monkeypatch):
+@pytest.mark.parametrize("on_github", [True, False])
+def test_execution_on_project_dir(cli_runner, project_tree, monkeypatch, on_github):
     """
     :type cli_runner: click.testing.CliRunner
     :type project_tree: py.path.local
     :type monkeypatch: _pytest.monkeypatch
     """
+    if on_github:
+        monkeypatch.setenv("GITHUB_WORKSPACE", "some/path")
+        expected_prefix = "::group::"
+    else:
+        expected_prefix = ""
     monkeypatch.chdir(project_tree.join("root_b"))
     command_args = ["-v", "--", "python", "-c", '"name: {name}"']
     result = cli_runner.invoke(deps_cli.cli, command_args)
     assert result.exit_code == 0, result.output
     matcher = LineMatcher(result.output.splitlines())
     matcher.fnmatch_lines(
         [
-            "dep_z (1/4)",
+            f"{expected_prefix}dep_z (1/4)",
             'deps: executing: python -c "name:\\ dep_z"',
             "deps: from:      *[\\/]test_projects0[\\/]dep_z",
             "deps: return code: 0",
-            "dep_b.1.1 (2/4)",
+            f"{expected_prefix}dep_b.1.1 (2/4)",
             'deps: executing: python -c "name:\\ dep_b.1.1"',
             "deps: from:      *[\\/]test_projects0[\\/]bs[\\/]dep_b.1.1",
             "deps: return code: 0",
-            "dep_b.1 (3/4)",
+            f"{expected_prefix}dep_b.1 (3/4)",
             'deps: executing: python -c "name:\\ dep_b.1"',
             "deps: from:      *[\\/]test_projects0[\\/]bs[\\/]dep_b.1",
             "deps: return code: 0",
-            "root_b (4/4)",
+            f"{expected_prefix}root_b (4/4)",
             'deps: executing: python -c "name:\\ root_b"',
             "deps: from:      *[\\/]test_projects0[\\/]root_b",
```

Comment:
Just a comment/question.

I recently I did somewhere else:
```
    matcher.fnmatch_lines(
        textwrap.dedent(
            '''\
            foo
                bar\
            '''
        ).splitlines()
    )
```
I did like to avoid all those `",`.
Do you think that is sloppy?

Reply:
Hmm not in particular.